WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldEditorView.swift

```diff
@@ -98,6 +98,13 @@ struct CustomFieldEditorView: View {
         }
         .background(Color(.listBackground))
         .toolbar {
+            ToolbarItem(placement: .topBarLeading) {
+                Button {
+                    presentationMode.wrappedValue.dismiss()
+                } label: {
+                    Text("Cancel") // todo-13493: set String to be translatable
```

Comment:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/14175/commits/255d9acc4050b4526faa298e11633a2930e307ea

Just out of caution in case we want to update the text, which had happened several times in this project already.